src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorBuildStep.groovy

```diff
@@ -45,6 +45,8 @@ class IqPolicyEvaluatorBuildStep
 
   String iqApplication
 
+  IqApplication iqApplicationV2
```

Comment:
Minor(ish): My preferred approach would be to keep the current field name, switch the type of IqApplication, and implement an XStream converter. Imagine this changing again in the future and an ever extending number of fields to maintain and Deprecate. Something like:

```
@@ -34,6 +33,17 @@ abstract class IqApplication
     return Jenkins.getInstance().getDescriptorOrDie(this.getClass())
   }

+  static final class ConverterImpl
+      extends AbstractSingleValueConverter {
+    boolean canConvert(Class clazz) {
+      return clazz.equals(IqApplication.class)
+    }
+
+    Object fromString(String str) {
+      return new SelectedApplication(str)
+    }
+  }
```

If the IQ team decides they'd rather maintain multiple fields, I'd suggest looking through best tips here [0] in particular around marking old fields transient. Lastly either approach should have some documentation to allow future contributors to understand why there is a Converter/multiple fields.

[0] https://wiki.jenkins.io/display/JENKINS/Hint+on+retaining+backward+compatibility

Reply:
> switch the type of IqApplication, and implement an XStream converter

Changed 32b411f